### ISSUE TITLE
Empire: Settlements get attacked and destroyed without a warning

### ISSUE THREAD

**Issue by Professor-Sword** [hyper1on/Empire-1_5-Continued-#3]
Not sure if it's a bug or something wrong with the military system. Usually whenever I place a settlement and it gets attacked, there's always a letter stating it will be attacked, and whatever other settlements I have will join military forces with that one. But nowadays, I just get a message that my settlement always loses, has its level drop, and have a building turned to rubble. I have a feeling it's because it's the only settlement with a military level of 2, because my other settlements of military levels 3 and 4 don't have this

TLDR: My settlements get attacked without warning

**Comment by Tazgul** [comment]
Do you by any chance have RimWar installed as well? The settlement defense only happens on Empire mod related attacks on your settlements, the RimWar related attacks are handled by RimWar itself, there your RimWar settlement score counts, not defense levels and you dont get the warning.

**Comment by patrick-mackay** [comment]
There is also an option in the rimwar configuration that can warn you when an enemy is approaching, and you can set the maximum number of tiles. However, it only works for walking warbands. There is nothing you can do to defend against flying warbands (drop pods).
